src/app/loadout-builder/NoBuildsFoundExplainer.tsx

```diff
@@ -207,7 +207,7 @@ export default function NoBuildsFoundExplainer({
   // two non-solar combat mods, mod assignment is trivially infeasible and we
   // can point that out directly?
 
-  const anyStatMinimums = Object.values(statFilters).some((f) => !f.ignored && f.min > 0);
+  const anyStatMinimums = statConstraints.some((f) => Boolean(f.minTier));
```

Comment:
If we switch to including ignored stats in `StatConstraints` I'll kind of need to undo a lot of these changes...